bite-size-notes/week-4-d-Example of LQR-name.md

```diff
@@ -0,0 +1,350 @@
+# Objectives: 
+1. Motivation of LQR
+2. Summarize the variation of LQR and corresponding Recati equation and optimal control. 
+3. Using ode45 to solve Reccati equation.
+# Introduction: 
+LQR (Linear Quatratic Regulater) is the a optimal solution to a full state feedback control to minimine the state error and control efforts. Let's break down this sentence to understand what is LQR ? 
+## What is a fullstate feedback control.
+We want to stabilize a system and drive all state to 0. A standard full state feed back control can achieve this goal.(There are other variation which are out of the scope) \
+A standar full feedback controle has the block diagram. 
+ ![fullfeedback](https://github.com/p8410077/AA548-spr2024/assets/11802603/bea8ed3a-5062-4d60-93c9-b213d4d856d5)
```

Comment:
I put the K gain on the bottom, but did not add the reference input, it might be reasonable to include reference, but the resource I refer did not specify there is reference input